test/e2e/proxy_test.go

```diff
@@ -200,6 +195,7 @@ func TestOAuthProxyE2E(t *testing.T) {
 
 	t.Logf("test image: %s, test namespace: %s", image, ns)
 
+	backendImage := "nginxdemos/nginx-hello:plain-text"
```

Comment:
just curious about the reason to replace the image?

Reply:
I was looking for a simple image to use and did not want to figure out how to build an image as a part of an e2e test and then upload it to the cluster's registry.

I'm open for ideas.